packages/yadm/build.sh

```diff
@@ -0,0 +1,25 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/TheLocehiliosan/yadm
+TERMUX_PKG_DESCRIPTION="Yet Another Dotfiles Manager"
+TERMUX_PKG_VERSION=1.12.0
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SHA256=c3d612d01e2027d5f457e0f7d120bc67251b716c373d99fe70638bd86edf107f
+TERMUX_PKG_SRCURL=https://github.com/TheLocehiliosan/yadm/archive/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_BUILD_IN_SRC="yes"
+TERMUX_PKG_DEPENDS="git"
+
+termux_step_make() {
+	:
+}
+
+termux_step_make_install () {
+    sed -i -E "1s@^#\!(.*)/[sx]?bin/(.*)@#\!/data/data/com.termux/files/usr/bin/\2@" $TERMUX_PKG_SRCDIR/yadm
+	cp $TERMUX_PKG_SRCDIR/yadm $TERMUX_PREFIX/bin/yadm
```

Comment:
You can omit the sed'ding, build-package.sh already does this, see https://github.com/termux/termux-packages/blob/2d93ba823e31b358926ff6ecd80975fc687244c2/build-package.sh#L1121